### DISCUSSION TITLE
Set org in scope based by name, not by id

### DISCUSSION THREAD

**Discussion by slabiak** [zitadel/zitadel#7240] [Q&A]
Currently to set org in scope while creating auth url we need to add sth like this `urn:zitadel:iam:org:id:249800091320218895`. Common case in multitenancy apps is that we have many sub domains for each tenant, lets say org1.myapp.com, org2.myapp.com. Now, in our SPA app, let's say React app, when user enters org1.myapp.com we detect that there is no access token present and we need to initate login process to org1. We need to create auth url on UI side, what we have is that we know org name (from domain, which is org1), and we need to set this org as scope. The issue is we dont have this mapping that org1 is 249800091320218895. So what we need to do is that we need to make additional call to backend to ask what is the orgId for org1. I think it would be much better if we can set scope like `urn:zitadel:iam:org:name:org1` - is sth similar possible or you plan to do such thing? I can't find abything about it in docs.

**Comment by hifabienne** [comment]
We do have two different scopes. The organization id scope and the primary domain scope.
In that case you can send the primary domain of the organization.
An organization name scope is currently not implemented and not planed:^.

https://zitadel.com/docs/apis/openidoauth/scopes#reserved-scopes
